main.js

```diff
@@ -1,13 +1,111 @@
+  const movieDescription = document.getElementById("display-info");
+  const reviewForm = document.getElementById("user-review");
+  const reviewsList = document.getElementById("reviews-list");
+  const resetReviews = document.getElementById("reset-reviews");
+  const peopleList = document.getElementById("people-list");
+  const peopleButton = document.getElementById("show-people");
 
+  fetch("https://ghibliapi.herokuapp.com/films")
+    .then((event) => {
+      return event.json();
+    })
+    .then((filmJson) => {
+      console.log(filmJson);
+      filmArr = filmJson;
 
+      populateOptions(filmJson);
 
+      titleDropdown.addEventListener("change", (event) => {
+        genr8MovieDesc(event.target.value);
+      });
 
+      peopleButton.addEventListener("click", (event) => {
+        fetch("https://ghibliapi.herokuapp.com/people")
+          .then((event) => {
+            return event.json();
+          })
+          .then((event) => {
+            for (let person of event) {
+              for (let film of person.films) {
+                if (film.includes(titleDropdown.value)) {
+                  const personName = document.createElement("li");
+                  personName.innerText = person.name;
+                  peopleList.append(personName);
+                }
+              }
+            }
+          });
+      });
 
-// To ensure Cypress tests work as expeded, add any code/functions that you would like to run on page load inside this function
+      reviewForm.addEventListener("submit", (event) => {
+        const reviewText = document.getElementById("review");
+        event.preventDefault();
+        const currentMovie = document.getElementById("selected-movie-title");
+        const reviewEntry = document.createElement("li");
 
-function run() {
- // Add code you want to run on page load here
+        if (currentMovie) {
+          if (reviewText.value) {
+            reviewEntry.innerHTML = `<strong>${currentMovie.innerText}:</strong> ${reviewText.value}`;
+            console.log(reviewEntry);
+            reviewsList.append(reviewEntry);
+            reviewText.value = "";
+          } else {
+            window.alert("Please write a review before attempting to submit.");
+          }
+        } else {
+          window.alert("Please select a movie first");
+        }
+      });
+
+      resetReviews.addEventListener("click", (event) => {
+        reviewsList.innerHTML = "";
+      });
+    });
+
+  function populateOptions(filmArr) {
+    for (let film of filmArr) {
+      const dropdownOption = document.createElement("option");
+      dropdownOption.setAttribute("value", `${film.id}`);
+      dropdownOption.innerText = film.title;
+      titleDropdown.append(dropdownOption);
+    }
+  }
+
+  function genr8MovieDesc(filmID) {
+    if (!filmID) {
+      movieDescription.innerHTML = "";
+    } else {
+      fetch(`https://ghibliapi.herokuapp.com/films/${filmID}`)
```

Comment:
Not really necessary since you already have the movie object from getting them all, but not gonna ding you on it